TSPL.docc/LanguageGuide/AccessControl.md

```diff
@@ -44,17 +45,36 @@ A *source file* is a single Swift source code file within a module
 Although it's common to define individual types in separate source files,
 a single source file can contain definitions for multiple types, functions, and so on.
 
+A *package* is a group of modules that you develop as a unit.
+You define the modules that form a package
+as part of configuring the build system you're using,
+not as part of your Swift source code.
+For example, if you use Swift Package Manager to build your code,
+you define a package in your `Package.swift` file
+using APIs from the [PackageDescription][] module;
+if you use Xcode, you specify the package name
+in the Package Access Identifier build setting.
+
+[PackageDescription](https://developer.apple.com/documentation/packagedescription)
+
 ## Access Levels
 
-Swift provides five different *access levels* for entities within your code.
+Swift provides six different *access levels* for entities within your code.
 These access levels are relative to the source file in which an entity is defined,
-and also relative to the module that source file belongs to.
+the module that source file belongs to,
+and the package that the module belongs to.
 
 - *Open access* and *public access*
   enable entities to be used within any source file from their defining module,
   and also in a source file from another module that imports the defining module.
   You typically use open or public access when specifying the public interface to a framework.
   The difference between open and public access is described below.
+- *Package access*
+  enables entities to be used within
+  any source files from their defining package,
```

Comment:
The comma is grammatically incorrect here because "not in any source file..." is a sentence fragment. 
```suggestion
  any source files from their defining package
```

Reply:
Changed.